index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<head></head>
+<body>
+    <h1>Haibin Yu</h1>
+
+    <h2>About</h2>
+
+    <p>
+        I was born in China and raised in Mexico. I moved to the United States as an unaccompanied child at the age of 16, in 2016, with the goal 
+        of acquiring a better education and professional opportunities. After years of hard work, and thanks to the help of the great people 
+        I have had around me, I managed to graduate from UC Berkeley with a degree in Computer Science last May of 2023. Due to my immigration status, 
+        I have struggled to land a job, but I am currently working on acquiring the necessary skills to become a full stack developer and work 
+        as an independent contractor or join the CTD Apprenticeship Program. In my free time I like to hang out with friends, work out, play video games, 
+        and explore the city.
+    </p>
+
+    <h2>Experience</h2>
+
+    <ul>
+        <li>
+            UCB USP Program - Web Developer: Created two apps through Google Apps Script and its Google Sheets API to retrieve and retrieve information 
+            from Google Sheets.
+        </li>
+        <li>
+            CARECEN Nonprofit - Immigration Law Legal Assistant: Help clients with different immigration applications: Work Permits, Status Adjustments, 
+            Naturalizations, etc.
+        </li>
+        <li>RIT (Refugee and Immigrant Transitions) - Highschool and Middleschool Tutor</li>
+    </ul>
+
+    <h2>Connect</h2>
+
+    <ul>
+        <li>
+            <a href="https://www.instagram.com/haibny/" target="_blank">Instagram</a>
+        </li>
+        <li>
+            <a href="https://www.linkedin.com/in/haibinyu/" target="_blank">LinkedIn</a>
+        </li>
+        <li>
+            <a href="https://github.com/haibn" target="_blank">GitHub</a>
+        </li>
+    </ul>
+</body>
```

Comment:
I think all of these lines need to be wrapped `<html> ... </html>` tags.

Reply:
I totally forgot about that. Thanks for the reminder!